verify-json.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Calling git status to trick git into realizing that no change has been made
+git status > /dev/null
```

Comment:
its possibly an idea in the wrong direction, but running ./verify-targets.json writes out a fresh `targets-config.json`, so when `git diff-index HEAD` is run, it reports that the file is modified, because its inspecting the timestamp. In testing, running `git status` causes it to actually check the file.

Try: `node verify-targets.js ; git diff-index HEAD`

compared to `node verify-targets.js ; git status > /dev/null; git diff-index HEAD`

As an alternative, I could have `verify-targets.js` see if the next argument is `ci`, and then read the existing file, and compare. I'd have to investigate specifics. I'm assuming I can't just read

I should also test this on windows and see if I need to `sort()` anything

Reply:
Ah yes, I see. A lot of the plumbing commands like `diff-index` only operate on metadata, so this will happen. Not sure if using `diff-index` for the test is ideal, we can probably just use `diff`.